.github/workflows/ci.yml

```diff
@@ -56,6 +56,8 @@ jobs:
               - 13-migration-entity-resolution
               - 20-export-tests
               - 21-role-permission-tests
+              - 21-large-file-derivatives
```

Comment:
I'm trying to understand the underlying intent of testing large file derivatives:  What is the purpose of the nightly large file derivatives build, if those tests are also run on every PR and merge into `development`?

Reply:
It's listed here so that they'll run when the PR is issued.  It would be removed in a follow-up PR after this PR is merged.

As an aside, I'm working on a PR that'll clean up the use of matrix and make it more useful for nightlies, PRs, and used by idc-isle-buildkit.